src/client/components/app-header/app-header.vue

```diff
@@ -1,61 +1,255 @@
 <template>
   <header role="banner" class="app-header">
-    <nuxt-link :to="localePath('index') + '/'">
-      <h1>Lean Web Kit</h1>
+    <nuxt-link :to="localePath('index')" class="app-header__identity">
+      <img class="app-header__logo" src="/images/logo.svg" alt="" />
+      <h1 class="app-header__title">{{ appTitle }}</h1>
     </nuxt-link>
-    <a :href="`#${contentId}`">{{ $t('skip_to_content') }}</a>
-    <nav>
+    
+    <a class="a11y-sr-only" :href="`#${contentId}`">{{ $t('skip_to_content') }}</a>
+    
+    <nav class="app-header__navigation" :class="{ 'app-header__navigation--open' : showNavigation }">
       <h2 class="a11y-sr-only">{{ menu.title }}</h2>
-      <ul class="inline-list">
-        <li v-for="item in menu.items" :key="item.slug">
+      
+      <ul class="app-header__inline-list">
+        <li class="app-header__list-item" v-for="item in slicedMenu" :key="item.slug">
           <nuxt-link :to="localePath({ name: 'slug', params: { slug: item.slug } })"
             class="app-header__menu-link"
           >
             {{ item.title }}
           </nuxt-link>
         </li>
       </ul>
-      <language-selector :locales="$i18n.locales" />
+      
+      <button class="app-header__action-button" v-if="menu.callToAction" @click="handleClick">{{ menu.callToAction.title }}</button>
+      <language-selector v-if="$i18n.locales" :locales="$i18n.locales" />
     </nav>
+    
+    <button class="app-header__toggle-menu" @click="toggleNavigation" >
+      <span class="a11y-sr-only">Toggle navigation</span>
+      <svg class="app-header__menu-icon" :class="{ 'is-open': showNavigation }" aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 512 512">
```

Comment:
Move this into a separate `menu-icon` component. It's cluttering up the `app-header` component and you are running out of names to BEM. So rather something like:
```vue
<template>
<svg class="menu-icon" :class="{ 'menu-icon--open': isOpen }" ...>
  <g>
    <path class="menu-icon__line--top">
    ...
  </g>
</svg>
</template>

<script>
export default {
 props: {
   isOpen: {
      type: Boolean,
      default: false,
    }
  }
}
</script>
```